.github/workflows/examples-works.yml

```diff
@@ -0,0 +1,20 @@
+name: VM
+on:
+  push:
+    branches: [examples-in-ci]
+
+jobs:
+  test-examples:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: 'npm'
+
+      - run: npm i
+      - run: npm run examples:ci
```

Comment:
I know you're probably just testing with this file, but when the time is right I can help add the "npm run examples:ci" line to vm-pr.yml and vm-build.yml

Can we also do a test if the example fails or errors out that it causes the ci step to fail?

Reply:
@ryanio I would assume that it is rather meant to be set up in this way by Cesar? 🤔

So, we have the options here to run the examples through this dedicated extra file or integrate in the existing CI files per package as Ryan mentioned. I have actually no strong opinion here, seems to work well in both ways.

Reply:
Short nit: in case we keep here, can we remove to just `examples.yml`?

Reply:
Indeed this was just for testing purposes - I was not sure where to add the script example

> add the "npm run examples:ci" line to vm-pr.yml and vm-build.yml

At first, I thought of doing so, but with this script we aim to run all existing examples in the repository, aiming at different packages

Personally, I would go with this dedicated script to run all the examples but open to suggestions :-D

Reply:
Hey guys after thinking a bit more about this I think I will move forward with creating the `examples.yml` file in the workflow folder - Let me know if we should proceed otherwise